wolfcrypt/src/wc_port.c

```diff
@@ -348,6 +348,59 @@ wolfSSL_Mutex* wc_InitAndAllocMutex(void)
     return m;
 }
 
+#ifdef USE_WOLF_STRTOK
+/* String token (delim) search. If str is null use nextp. */
+char* wc_strtok(char *str, const char *delim, char **nextp)
```

Comment:
Good point! I was going off the comments in types.h where it says "use only Thread Safe version of strtok". Since there is no mutex protection here I assumed there could be scenario for that.

Personally, I don't see where we'd ever have a threading issue.